pkg/apis/admission/types.go

```diff
@@ -56,6 +56,27 @@ type AdmissionRequest struct {
 	// "binding", and kind "Binding".
 	// +optional
 	SubResource string
+
+	// RequestKind is the type of object being manipulated by the the original API request.  For example: Pod
+	// If this differs from the value in "kind", an equivalent match and conversion was performed.
+	// See documentation for the "matchPolicy" field in the webhook configuration type.
+	// +optional
+	RequestKind *metav1.GroupVersionKind
+	// RequestResource is the name of the resource being requested by the the original API request.  This is not the kind.  For example: pods
+	// If this differs from the value in "resource", an equivalent match and conversion was performed.
+	// See documentation for the "matchPolicy" field in the webhook configuration type.
+	// +optional
+	RequestResource *metav1.GroupVersionResource
+	// RequestSubResource is the name of the subresource being requested by the the original API request.  This is a different resource, scoped to the parent
```

Comment:
nit: misleading. Above resource = GVR. here resource = just the subresource without version and group.

Reply:
updated examples and added "fully-qualified" to the kind and resource field doc